scripts/202306xx_populate_paymenttype_default/add_fixed_payment_type_default.js

```diff
@@ -4,31 +4,87 @@
 Add PaymentType.Fixed to all forms with existing payments_field object
 */
 
-// Count number of forms that contains payments_field property
-db.getCollection('forms')
-  .find({ payments_field: { $exists: true } })
-  .count()
+// Batch 1 2022
+const queriesModifiedIn2022 = {
+  payments_field: { $exists: true },
```

Comment:
Is there a need for this check when line 34 already contains the `payments_field` check?

Reply:
Ooops, should be removed! Thanks for catching